src/Task/PhpCsFixerV2.php

```diff
@@ -72,7 +73,13 @@ public function run(ContextInterface $context)
         $arguments->addOptionalArgument('--allow-risky=%s', $config['allow_risky'] ? 'yes' : 'no');
         $arguments->addOptionalArgument('--cache-file=%s', $config['cache_file']);
         $arguments->addOptionalArgument('--config=%s', $config['config']);
-        $arguments->addOptionalCommaSeparatedArgument('--rules=%s', $config['rules']);
+
+        if (is_array($rules = $config['rules']) && ArrayType::isMap($rules)) {
```

Comment:
Perhaps we can remove the array check in that case. Since this is a cutting edge feature they have not provided any documentation yet, so the best I can do is link the [PR that introduced the change](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2494), which I also linked from the issue requesting this feature (#288).